packages/config/src/utils/extensions/utils.ts

```diff
@@ -0,0 +1,242 @@
+import { NetlifyAPI } from 'netlify'
+
+import { JIGSAW_URL } from './constants.js'
+
+export type Extension = {
+  id: string
+  name: string
+  slug: string
+  hostSiteUrl: string
+  installedOnTeam: boolean
+}
+
+export const getInstalledExtensionsForSite = async ({
+  accountId,
+  siteId,
+  netlifyToken,
+}: {
+  accountId: string
+  siteId: string
+  netlifyToken: string
+}) => {
+  const url = new URL(
+    `${JIGSAW_URL}/team/${encodeURIComponent(accountId)}/integrations/installations/${encodeURIComponent(siteId)}`,
+    JIGSAW_URL,
+  )
+  const extensionsResponse = await fetch(url.toString(), {
```

Comment:
So this is a second call we'll need to make to Jigsaw on every build? If I understand it right, we're asking Jigsaw 1) what extensions need to be installed automatically and 2) what extensions are installed for a given site. We'll then match up these two pieces of information and decide which extensions need to be installed. Have we considered merging this into one single call and letting Jigsaw tell us the extensions that need to be installed based on both factors? Or is there any other place where this call is useful? Just worried about introducing more points of failure and latency to builds, with not one but two API calls.

Reply:
👏 thanks for the feedback! I removed these extra calls to jigsaw.